modules/ROOT/examples/live-demos/typography/index.js

```diff
@@ -0,0 +1,49 @@
+tinymce.init({
+  selector: "textarea#advanced-typography",
+  height: 300,
+  menu: { tools: { title: 'Tools', items: 'code wordcount typography' }},
+  plugins: 'code typography',
+  toolbar: 'bold italic | code typography',
+  content_langs: [
+    { title: 'English', code: 'en' },
+    { title: 'Spanish', code: 'es' },
+    { title: 'German', code: 'de' }
```

Comment:
```suggestion
```
Don't think we need this option as it is only used by the `language` toolbar button which is already removed